scripts/install_opengv.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+git clone https://github.com/laurentkneip/opengv.git "${temp_dir}/opengv"
+
+cmake \
+	-S "${temp_dir}/opengv" \
+	-B "${temp_dir}/opengv/build" \
+    -DEIGEN_INCLUDE_DIR="${temp_dir}/gtsam/gtsam/3rdparty/Eigen" \
+    -DEIGEN_INCLUDE_DIRS="${temp_dir}/gtsam/gtsam/3rdparty/Eigen"
```

Comment:
Minor, but use same format and indentation as other files for consistency. So `-D <FLAG>` and not `-D<FLAG>`.